[Flaky Test] Filebeat test_registrar.Test.test_clean_removed_with_clean_inactive

## Flaky Test

* **Test Name:**  test_registrar.Test.test_clean_removed_with_clean_inactive
* **Branch:** master
* **Artifact Link:** https://beats-ci.elastic.co/job/elastic+beats+pull-request+multijob-linux/4809/beat=filebeat,label=ubuntu/testReport/junit/test_registrar/Test/test_clean_removed_with_clean_inactive/

### Artifcats

Below is a zip file with the Artifacts from the build:

[test_registrar.Test.test_clean_removed_with_clean_inactive.zip](https://github.com/elastic/beats/files/2835784/test_registrar.Test.test_clean_removed_with_clean_inactive.zip)

this is also still happening on osx: https://travis-ci.org/elastic/beats/jobs/527412635

```
======================================================================
FAIL: Checks that files which were removed, the state is removed
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/travis/gopath/src/github.com/elastic/beats/filebeat/tests/system/test_registrar.py", line 924, in test_clean_removed_with_clean_inactive
    assert count == 1
AssertionError: 
-------------------- >> begin captured stdout << ---------------------
render config
registry size: 2
registry size after remove: 2
--------------------- >> end captured stdout << ----------------------
```

--- a/filebeat/tests/system/test_registrar.py
+++ b/filebeat/tests/system/test_registrar.py
@@ -820,7 +820,6 @@ class Test(BaseTest):
         # Make sure the last file in the registry is the correct one and has the correct offset
         assert data[0]["offset"] == self.input_logs.size(file3)
 
-    @unittest.skipIf(os.name == 'nt', 'flaky test https://github.com/elastic/beats/issues/7690')
     def test_clean_removed(self):
         """
         Checks that files which were removed, the state is removed
@@ -870,7 +869,6 @@ class Test(BaseTest):
         # Make sure the last file in the registry is the correct one and has the correct offset
         assert data[0]["offset"] == self.input_logs.size(file2)
 
-    @unittest.skip('flaky test https://github.com/elastic/beats/issues/10606')
     def test_clean_removed_with_clean_inactive(self):
         """
         Checks that files which were removed, the state is removed

